wordpress-mobile/release-toolkit-gutenberg-mobile
# Add create GBM PR flow

This adds a flow to create a GBM PR.  

This adds
`pkg/gbm/gbm.go`  to create a GB linked GBM PR
`pkg/release/gbm.go`  to create a release specific GBM PR

The PR also includes some updates to the `prepare` script to create all release PRs. 
See `go run main.go release prepare -h` to see the relevant flags to create the additional PRs.

Testing:
- Set the env to point to a test Github org
- run `go run main.go release prepare 0.1.0 --gbm --integrate` 
- Wait for prompts to create each of the 4 release PRs.
- Check the GB and GBM PRs to verify that the all release changes are performed and that the PR descriptions are accurate. Note this should fill in all of the GBM release information.
-
